Source/Core/VideoBackends/Vulkan/VulkanContext.cpp

```diff
@@ -679,9 +744,8 @@ bool VulkanContext::CreateDevice(VkSurfaceKHR surface, bool enable_validation_la
   // Enable debug layer on debug builds
```

Comment:
It wasn't added by this PR, but I don't think this comment makes sense or ever made sense; even in 77a128ab8707659a531029a21a2e874108eedf38 it looks like it used `g_Config.bEnableValidationLayer;` and was presumably configurable at runtime.